### DISCUSSION TITLE
how do i make it look like i bought something, like an amazon item

### DISCUSSION THREAD

**Discussion by Smokey101** [DSJAS/DSJAS#49] [General Q&A] [answered]
what the title said ^

**Comment by ejv2** [comment, marked as answer]
Take a look at [BANK URL]/admin/bank/accounts.php#forceTransaction and see if that describes what you mean. Replace [BANK URL] with whatever address you are using with the bank.

**Reply by Smokey101** [reply]
also another question, how do i put this into my vm, the ipv4 gateway dosent work and if i type localhost into browser it still dont work

**Reply by ejv2** [reply]
Try getting the ip of your host with ``ipconfig`` (or ``ifconfig`` on Mac etc.). Use that as the IP in the vm and don't forget the port number. If you want to disguise this, try following the instructions on [URL spoofing](https://github.com/DSJAS/DSJAS/blob/master/docs/administration/A%20Guide%20to%20URL%20Spoofing.md).

**Reply by Smokey101** [reply]
Im really sorry for asking 4 questions, but how do i set my port number to the same of my vm?

**Reply by ejv2** [reply]
Sorry, I don't think I understand. Please explain what you mean by the port number to the same os your vm.

**Reply by Smokey101** [reply]
in the 2nd reply you said "dont forget the port number", what is the port number, btw im using mamp this might be why its not working

**Reply by ejv2** [reply]
After the IP address, put a colon and then the port number that you configured your server to listen at. So, if the IP was 192.168.0.5 and the port was 8080, your address would be 192.168.0.5:8080. The port number will be in the MAMP settings panel and is the same in the guest VM.

**Reply by Smokey101** [reply]
Thanks alot for responding, however i dont know where to put the port and ip, as before instead i just put the same ip settings on my vm as my main machine, ill try to do this myself, but thank you alot.

**Reply by ejv2** [reply]
I think you might be a bit confused. You only need to run the server on your host (outside your vm) and do not need to run it inside the vm. Then, you navigate to the browser using \<IP of host\>:\<server port\> (e.g: 127.0.0.1:8080). If you want to set up a spoofed custom domain, you need to edit the [hosts file](https://github.com/DSJAS/DSJAS/blob/master/docs/administration/A%20Guide%20to%20URL%20Spoofing.md).

**Reply by Smokey101** [reply]
OMG thank you so much, it finally works
